src/java/org/apache/cassandra/service/consensus/TransactionalMode.java

```diff
@@ -112,6 +112,10 @@ public ConsistencyLevel commitCLForStrategy(ConsistencyLevel consistencyLevel)
         return consistencyLevel;
     }
 
+    // TODO (required): This won't work for migration directly from none to full because there is no safe system to read from
```

Comment:
What a migration from `none` to `full` effectively ran accord in `mixed_reads` mode during migration?

Reply:
Yeah that is what it would need to do and it's what I implemented later when splitting into two phases. `TransactionalMode` takes in a `ClusterMetadata` and `Token` and makes a determination about `ConsistencyLevel` that is specific to whether the range is the `Token` is in has completed migration so it runs the appropriate interop `ConsistencyLevel` until it is no longer needed.